WinRT.Runtime/Projections/Type.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace ABI.System
 {
-    [WindowsRuntimeType]
+    [WindowsRuntimeType("Windows.UI.Xaml")]
```

Comment:
Is this accurate?

Reply:
oops, meant to fix that - should be Windows.UI.Xaml.Interop

Reply:
This should be the assembly right not the namespace?

Reply:
oh, had it right to begin with - yes, the assembly.  e.g., C:\Windows\System32\WinMetadata\Windows.UI.Xaml.winmd